Chocolatey.Cake.Recipe/Content/credentials.cake

```diff
@@ -52,6 +52,25 @@ public class PackageSourceCredentials
     }
 }
 
+public class DockerSourceCredentials
+{
+    public string Server { get; private set; }
+    public string User { get; private set; }
+    public string Password { get; private set; }
+
+    public bool HasCredentials
+    {
+        get { return !string.IsNullOrEmpty(User) && !string.IsNullOrEmpty(Password); }
+    }
+
+    public DockerSourceCredentials(string user, string password, string server = "dockerhuburlhere?")
```

Comment:
We agreed that this should default to Docker Hub, right?

Reply:
Or, we simply remove the default, as it will likely get overwritten by the empty environment variable, if it doesn't exist.

Reply:
Done!